elasticdl/python/worker/worker.py

```diff
@@ -513,9 +518,17 @@ def _process_save_model_task_if_needed(self):
         task, dataset = (
             self._task_data_service.get_save_model_task_and_dataset()
         )
-        if task and dataset:
+        if (task is not None) and (dataset is not None):
             # TODO: Implement the save model execution process
-            return
+            config_dict = {
+                pair.key: pair.value for pair in task.extended_config
+            }
+            saved_model_path = config_dict.get(
+                SaveModelConfig.SAVED_MODEL_PATH
+            )
+            logger.info("The saved model path is {}".format(saved_model_path))
```

Comment:
-> The path to saved model is

Reply:
Good suggestion, update it to "The path to export model"